src/operations/common/GitHubRepoRef.ts

```diff
@@ -15,6 +16,18 @@ export const GitHubDotComBase = "https://api.github.com";
  */
 export class GitHubRepoRef extends AbstractRepoRef {
 
+    public static for(params: { owner: string, repo: string, sha?: string, rawApiBase?: string, path?: string, branch?: string }): GitHubRepoRef {
```

Comment:
Not keen on using a reserved word as a function name even if it is legal

Reply:
oh good call